man1/aur-sync.1

```diff
@@ -162,6 +162,11 @@ Default: \fI$XDG_CACHE_HOME/aurutils/sync\fR
 Parent directory for temporary files.
 .RE
 
+.B AURURI
+.RS
+Sets the default URI for fetching packages.
```

Comment:
I wonder if we should generalize this variable to include the RPC. In particular, `aur-rpc` would also refer to it.